src/NosCore.Controllers/Inventory/UpgradePacketHandler.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using ChickenAPI.Packets.ClientPackets.Player;
+using ChickenAPI.Packets.Enumerations;
+using NosCore.Core.I18N;
+using NosCore.Data;
+using NosCore.Data.Enumerations.I18N;
+using NosCore.GameObject;
+using NosCore.GameObject.Helper;
+using NosCore.GameObject.Networking.ClientSession;
+using NosCore.GameObject.Providers.InventoryService;
+using NosCore.GameObject.Providers.ItemProvider.Item;
+using NosCore.GameObject.Providers.UpgradeService;
+using Serilog;
+
+namespace NosCore.PacketHandlers.Inventory
+{
+    public class UpgradePacketHandler : PacketHandler<UpgradePacket>, IWorldPacketHandler
+    {
+        private readonly ILogger _logger;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_logger' or refactor the code to use its value.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4518428)